doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -0,0 +1,115 @@
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+## Tier 2
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+ 
+## Tier 3
+ 
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+
+## Tier 4
+
+ | Tool Type | Tool Name                 | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ |           |                           |                               |                         |             |
+ 
+## Not yet classified
+ 
+ | Tool Type | Tool/API Name             | Regular Testing In Node.js CI | Integrated with Node.js | Target Tier |
+ |-----------|---------------------------|-------------------------------|-------------------------|-------------|
+ | FFDC      | node-report               | No                            | No                      |     1       |
+ | Memory    | llnode                    | ?                             | No                      |     2       |          
+ | Memory    | mdb_V8                    | No                            | No                      |     4       | 
+ | Memory    | node-heapdump             | No                            | No                      |     2       |
+ | Memory    | V8 heap profiler          | No                            | Yes                     |     1       |
+ | Memory    | V8 sampling heap profiler | No                            | Yes                     |     1       |
+ | AsyncFlow | Async Hooks(API)          | ?                             | Yes                     |     1       |
+ | Debugger  | V8 Debug protocol(API)    | No                            | Yes                     |     1       |
+ | Debugger  | Command line Debug Client | ?                             | Yes                     |     1       |
+ | Debugger  | Chrome Dev tools          | ?                             | No                      |     3       |
+ | Debugger  | Chakracore - time-travel  | No                            | data source only        | to early    |
```

Comment:
`data` -> `Data` (for consistency)?

Reply:
Not sure, but was `to early` intended to be `Too early`?

Reply:
yes and yes